h/schemas/forms/accounts/login.py

```diff
@@ -54,7 +54,7 @@ def validator(self, node, value):
             )
             raise err from exc
 
-        if user is None:
+        if user is None or user.deleted:
```

Comment:
I reckon this belongs here:

https://github.com/hypothesis/h/blob/main/h/services/user.py#L158

Is "fetch_for_login" after all.

Reply:
Done